arrow-libs/fx/arrow-fx-coroutines/src/commonMain/kotlin/arrow/fx/coroutines/OS.kt

```diff
@@ -0,0 +1,8 @@
+package arrow.fx.coroutines
+
+public expect object OS {
```

Comment:
How about putting this class in test sourceSet

Reply:
I tried that, but `nativeTest` does not work as `sourceSet` currently. So I would need to create 20x `test` source sets, one per platform